projects/eslint-plugin/rules/general/lib/rules/prefer-length-check.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const message = 'prefer using .length instead of .length === 0 or .length > 0';
+
+module.exports = {
+	create(context) {
+		return {
+			BinaryExpression(node) {
+				const {left, operator, right} = node;
+				const STRICT_EQUALITY = '===';
+				const MORE_THAN = '>';
+				const LENGTH = 'length';
```

Comment:
These can be moved up and out of this expression, that we only create these variables once.

If we leave them here, we essentially are creating these variables _everytime_ eslint crawls the AST and sees a BinaryExpression

Reply:
This one was a test for you, and you passed

<img src="https://media3.giphy.com/media/7YrnYstmGxYFa/giphy.gif"/>